### ISSUE TITLE
No Style for Disabled Buttons

### ISSUE THREAD

**Issue by martypowell** [baltimorecounty/baltimorecountymd.gov-assets#26]
This is the active tax sale. The next button is disabled, but appears to be visible. @danfox01 @sdurphy I would think this is pretty pressing, at least for tax sale.

![image](https://cloud.githubusercontent.com/assets/1933400/25617307/a739c582-2f0f-11e7-852b-5cf2ee28c8de.png)

**Comment by sdurphy** [comment]
This is the CSS to make the text gray for disabled buttons:
input[disabled] {
    color: #999;
}

**Comment by mxsnyder** [comment]
That will change every disabled input element. If you just want to target buttons, you have to target:
```CSS
input[type=button][disabled], input[type=submit][disabled], button[disabled] {
    color: #999;
}
```

**Comment by martypowell** [comment]
thanks @mxsnyder, was planning on updating that

**Comment by martypowell** [comment]
Fixed by #33
